Src/IMX/Api/src/Imx.Sdk.Gen/Imx.Sdk.Gen.csproj

```diff
@@ -6,21 +6,17 @@
     <AssemblyName>Imx.Sdk.Gen</AssemblyName>
     <PackageId>Imx.Sdk.Gen</PackageId>
     <OutputType>Library</OutputType>
-    <Authors>Immutable</Authors>
-    <Company>Immutable</Company>
-    <AssemblyTitle>Immutable C# Core SDK API Library</AssemblyTitle>
```

Comment:
@suluvai do these need to be updated manually?

Reply:
We can potentially look at updating the template file to include these automatically.

Reply:
Will create a task

Reply:
https://immutable.atlassian.net/browse/DX-1729